config/hubs/2i2c.cluster.yaml

```diff
@@ -15,8 +15,8 @@ hubs:
     config:
       docs_service:
         enabled: true
-        repo: https://github.com/2i2c-org/pilot-hubs
-        branch: gh-pages
+        repo: https://github.com/jupyterhub/nbgitpuller
+        branch: master
```

Comment:
Should not be the `gh-pages` branch as well? Otherwise, why we are not pointing to the master pilot-hubs?

Reply:
@yuvipanda @choldgraf ⬆️

Reply:
Haha great catch, @damianavila. https://github.com/2i2c-org/pilot-hubs/pull/408 fixes it